massa-api-exports/src/api_request.rs

```diff
@@ -0,0 +1,9 @@
+use crate::page::PageRequest;
+use serde::{Deserialize, Serialize};
+
+/// Wrap request params into struct for ApiV2 method
+#[derive(Deserialize, Serialize)]
+pub struct ApiRequest {
```

Comment:
As it's common and small, we could add it directly to `lib.rs` but may be in the future, we'll create `types.rs`or `common.rs`to keep `lib.rs`clean

Reply:
Done, 

If u want i can squash my commit on this branch